pennylane/ops/qubit.py

```diff
@@ -634,6 +634,15 @@ class CSWAP(Operation):
     def _matrix(cls, *params):
         return cls.matrix
 
+    @staticmethod
+    def decomposition(wires):
+        decomp_ops = [
+            qml.Toffoli(wires=[wires[0], wires[2], wires[1]]),
+            qml.Toffoli(wires=[wires[0], wires[1], wires[2]]),
+            qml.Toffoli(wires=[wires[0], wires[2], wires[1]]),
+        ]
+        return decomp_ops
```

Comment:
Nice!

Reply:
I hope to contribute with more improvements :D

Reply:
Once this is merged in, feel free to get started on any other of our UnitaryHACK labeled issues!